tests/fuzzer/fuzzer.py

```diff
@@ -233,14 +235,15 @@ def run(
     :param seed: first seed, incremented by 1 for each individual test
     """
     infra_graph = make_graph(base_url, scenario.infra)
+    post_with_timeout(editoast_url + f"infra/{scenario.infra}/load").raise_for_status()
```

Comment:
This one doesn't really need a timeout, it's expected to take much longer than other requests.

Which also means that we can now significantly lower the timeout value for the rest of the requests, but that can wait

```suggestion
    requests.post(editoast_url + f"infra/{scenario.infra}/load").raise_for_status()
```